File: src/main/java/org/jukeboxmc/item/behavior/ItemGoldenApple.java
package org.jukeboxmc.item.behavior;

import org.jukeboxmc.item.ItemType;
import org.jukeboxmc.util.Identifier;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemGoldenApple extends ItemFood {

    public ItemGoldenApple( Identifier identifier ) {
        super( identifier );
    }

    public ItemGoldenApple( ItemType itemType ) {
        super( itemType );
    }

    @Override
    public float getSaturation() {
        return 9.6f;
    }

    @Override
    public int getHunger() {
        return 4;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemGoldenApple.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.ItemType

class ItemGoldenApple(itemType: ItemType, countNetworkId: Boolean) : JukeboxItemFood(itemType, countNetworkId) {

    override fun getSaturation(): Float {
        return 9.6F
    }

    override fun getHunger(): Int {
        return 4
    }
}